src/topp/ConsensusID.cpp

```diff
@@ -763,7 +763,7 @@ class TOPPConsensusID :
               auto acciter_inserted = iter_inserted.first->second.emplace(hit.getAccession());
               if (acciter_inserted.second)
               {
-                prot_ids[mzml_to_new_run_idx[original_file]].getHits().emplace_back(std::move(hit));
+                prot_ids[mzml_to_new_run_idx[original_file]].getHits().emplace_back(hit);
```

Comment:
correct - can't move reference

Reply:
@timosachsenberg I disagree, `hit` is not a reference but of type `ProteinHit` so it can be moved and here I think it *should* be moved 

https://github.com/OpenMS/OpenMS/blob/8594c2ad50b1e3d3263235ea592ed02049dbcb11/src/openms/include/OpenMS/METADATA/ProteinIdentification.h#L333

Reply:
Then we need to check why clang things it can’t be moved